chat_shell/chat_shell/prompts/builder.py

```
@@ -279,6 +279,179 @@ def append_deep_thinking_prompt(system_prompt: str, enable_deep_thinking: bool)
+<content>
+```
+
+The `artifact_id` in parentheses is what you need to pass to `update_artifact`.
+
+**Modification requests include (修改请求包括):**
+- "扩充/扩写第X段" (Expand paragraph X)
+- "删除/移除第X章/段" (Delete chapter/paragraph X)
+- "修改/编辑这篇文章" (Modify/edit this article)
+- "把第X段改成..." (Change paragraph X to...)
+- "把这句话改成..." (Change this sentence to...)
+- "把XXX改下：...可以改成..." (Change XXX: ... can be changed to...)
```

Ruff (0.14.11)

String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)